src/services/meteorsService.js

```diff
@@ -0,0 +1,36 @@
+require("dotenv").config();
+const axios = require("axios");
+const https = require("https");
+
+const getMeteorsDataWithQuery = require("../utils/meteorsMapper");
+
+const httpsAgent = new https.Agent({
+  rejectUnauthorized: false
+});
+
+const START_DATE = "2024-03-22";
+const END_DATE = "2024-03-29";
```

Comment:
I don't see any place where you use this constants
only in logs

Reply:
Updated